plugins/package-managers/maven/src/funTest/assets/projects/synthetic/tycho/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.ossreviewtoolkit</groupId>
+    <artifactId>tycho-test-aggregator</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <name>[pom] Aggregator</name>
+    <packaging>pom</packaging>
+    <description>
+        This is a test project for the ORT Tycho package manager.
+    </description>
+    <url>http://maven.apache.org</url>
+    <scm>
+        <connection>scm:git:https://example.com/git/repository.git</connection>
+        <developerConnection>scm:git:https://example.com/git/repository.git</developerConnection>
+        <url>https://example.com/git/repository</url>
+    </scm>
+    <organization>
+        <name>The Apache Software Foundation</name>
+    </organization>
+    <developers>
+        <developer>
+            <organization>The Apache Software Foundation</organization>
+            <name>Dev A</name>
+        </developer>
+        <developer>
+            <organization>Google</organization>
+            <name>Dev B</name>
+        </developer>
+    </developers>
```

Comment:
Use fake names or ORT references as organization and developers, but not existing entities that are not associated with the project.

Reply:
This section was copied from the test project for Maven (assets/projects/synthetic/maven/pom.xml).

Reply:
Ok, we shouldn't use them their either. Mabye could it fix that in your PR and I make a separate one for the other place. Or if you want you could do both, as you prefer.

Reply:
I have replaced the organization names by fictive ones and added another commit which does the same for the Maven test projects. Not sure whether there are other places though that would be affected.